doc/doc.go

```diff
@@ -117,9 +117,13 @@ func outputs(list *ast.ObjectList) []Output {
 		if is(item, "output") {
 			name, _ := strconv.Unquote(item.Keys[1].Token.Text)
 
+			items := item.Val.(*ast.ObjectType).List.Items
```

Comment:
I think this might panic in some cases, can we do a check like:

```
var items []*ast.ObjectItem

if obj, ok := item.Val.(*ast.ObjectType); ok {
  items = obj.List.Items
}

```